bin/submitMG.sh

```diff
@@ -22,13 +33,17 @@ cd job
 echo "copying madgraph tarball ..."
 #cp -r /eos/experiment/fcc/hh/utils/generators/MG5_aMC_v2.6.1.tar.gz .
 #cp -r /eos/experiment/fcc/hh/utils/generators/MG5_aMC_v2.6.7.tar.gz .
-cp -r /eos/experiment/fcc/hh/utils/generators/MG5_aMC_v2.8.2.tar.gz .
+# cp -r /eos/experiment/fcc/hh/utils/generators/MG5_aMC_v2.8.2.tar.gz .
+cp -r /afs/cern.ch/work/b/bistapf/MG5_aMC_v2.5.5.tar.gz .
```

Comment:
I copied `MG5_aMC_v2.5.5.tar.gz` to `/eos/experiment/fcc/hh/utils/generators` so it is in some global space. Also can you not use `-r` --- you are copying just one file.